Detections/SecurityEvent/UserAccountAddedToPrivlegeGroup_1h.yaml

```diff
@@ -39,6 +41,32 @@ query: |
   | extend SimpleMemberName = substring(MemberName, 3, indexof_regex(MemberName, @",OU|,CN") - 3)
   | project TimeGenerated, EventID, Activity, Computer, SimpleMemberName, MemberName, MemberSid, TargetUserName, TargetDomainName, TargetSid, UserPrincipalName, SubjectUserName, SubjectUserSid
   | extend timestamp = TimeGenerated, AccountCustomEntity = SimpleMemberName, HostCustomEntity = Computer
+  ),
+  (
+  WindowsEvent 
+  // 4728 - A member was added to a security-enabled global group
+  // 4732 - A member was added to a security-enabled local group
+  // 4756 - A member was added to a security-enabled universal group
+  | where EventID in (4728, 4732, 4756)  and  not(EventData has "S-1-5-32-555")
+  | extend SubjectUserSid = tostring(EventData.SubjectUserSid)
+  | extend Account =  strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
+  | extend AccountType=case(Account endswith "$", "Machine", iff(SubjectUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", iff(isempty(SubjectUserSid), "", "User")))
```

Comment:
AccountType not corrected here.

Reply:
@oshezaf what you see here is outdated, in this link you can see that the AccountType is updated.
https://github.com/Azure/Azure-Sentinel/blob/e9850fdb367aa3b0f2b35890482cf64e197fbd0f/Detections/SecurityEvent/UserAccountAddedToPrivlegeGroup_1h.yaml